internal/pkg/service/buffer/definition/repository/branch_repo_test.go

```diff
@@ -29,8 +29,8 @@ func TestRepository_Branch(t *testing.T) {
 
 	// Fixtures
 	projectID := keboola.ProjectID(123)
-	branch1 := branchTemplate(key.BranchKey{ProjectID: 123, BranchID: 567})
-	branch2 := branchTemplate(key.BranchKey{ProjectID: 456, BranchID: 789})
+	branchKey1 := key.BranchKey{ProjectID: 123, BranchID: 567}
+	branchKey2 := key.BranchKey{ProjectID: 456, BranchID: 789}
 
```

Comment:
Unified, in the test main scope, there are defined only keys.
Entities are created/loaded in individual scopes.